bench-tps/src/cli.rs

```diff
@@ -1,6 +1,8 @@
 use {
     clap::{crate_description, crate_name, App, Arg, ArgMatches},
-    solana_faucet::faucet::FAUCET_PORT,
+    solana_clap_utils::input_validators::{is_url, is_url_or_moniker},
+    solana_cli::cli::CliConfig,
```

Comment:
I think ideally this would come from a shared `cli-utils` or something like that crate that both cli and bench-tps pull from.